pages/charcuterie.js

```diff
@@ -1,6 +1,11 @@
 import Head from 'next/head'
+import videoIcon from '../public/assets/logos/video-icon.svg'
+import githubIcon from '../public/assets/logos/github-icon.svg'
+import mediumIcon from '../public/assets/logos/medium-icon.svg'
+import badgeIcon from '../public/assets/logos/our-pick-badge.svg'
```

Comment:
What are your thoughts on passing in a string that's a key for the badge icon and then pulling these in the `ResourceCard` component? I'm fine either way since it's not like we'll be reusing this component a lot but an idea!

Reply:
ooh sorry not 100% sure what you mean by this. do you mean passing a string as a prop of ResourceCard, and then rendering the asset within ResourceCard based on the prop?

Reply:
my bad for the late reply! gonna merge for now and we can clarify this afterwards, open to changes :)

Reply:
Oh I meant for example you pass in a prop named `icon` which is a string and it's one of `[video, github, medium, badge]`. Then you import the icons inside the `ResourceCard` component. Was just a thought I had cause in the case you use this component elsewhere you have to import the icons again in the parent component.

But this was just a suggestion, absolutely not blocking. Please merge! ❤️